.github/workflows/release.yml

```diff
@@ -443,37 +443,22 @@ jobs:
         run: |
           cd ${{ github.workspace }}/bin
           yarn install && yarn link @pulumi/eks
+      - name: Generate go test Slice
+        id: test_split
+        uses: hashicorp-forge/go-test-split-action@v2.0.0
+        with:
+          working-directory: examples
+          flags: -tags=nodejs
+          total: ${{ matrix.parallel }}
+          index: ${{ matrix.index }}
       - name: Run tests
-        run: make specific_test TestName=${{ matrix.test-name }} LanguageTags=nodejs
+        run: cd examples && go test -tags=nodejs -v -json -count=1 -cover -timeout 3h -parallel 12 . --run="${{ steps.test_split.outputs.run}}" 2>&1 | tee /tmp/gotest.log | gotestfmt
     strategy:
       fail-fast: false
+      max-parallel: 10
```

Comment:
2/3 of parallelism

Reply:
Doesn't this mean the parallelism is 10*12?

Reply:
The max test parallelism is 10*12, yes. But there's only 2-3 tests per batch.

But you're right, when we'll eventually double our tests this would be an issue again. I'm gonna change this to 6. That should be a good middle ground.